database-commons/src/test/java/io/cdap/plugin/db/batch/DatabasePluginTestBase.java

```diff
@@ -91,4 +92,33 @@ protected void runETLOnce(ApplicationManager appManager,
     workflowManager.start(arguments);
     workflowManager.waitForRun(ProgramRunStatus.COMPLETED, 5, TimeUnit.MINUTES);
   }
+
+  /**
+   * Added to prevent 'Ambiguous method call' issue
+   */
+  protected static void assertObjectEquals(Object expected, Object actual) {
```

Comment:
looks like utility methods. Should not be inside base class

Reply:
Moved to [DatabasePluginTestUtil](https://github.com/vladhlinskiy/database-plugins/pull/9/files#diff-1b53824ce9efceea02173f9a604e1184R34).